app/controllers/medicaid/intro_marital_status_indicate_spouse_controller.rb

```diff
@@ -0,0 +1,61 @@
+module Medicaid
+  class IntroMaritalStatusIndicateSpouseController <
+    Medicaid::MemberStepsController
+    helper_method :spouse_options
+
+    def update
+      @step = step_class.new(step_params)
+
+      if @step.valid?
```

Comment:
I would replace:
```
def update
 ...
end
```

with:

```
    def after_successful_update_hook
      update_spouse_of_other_member_if_present
    end
```

especially since the controller_spec is not testing the non-valid code path here

Reply:
also this would fix the codeclimate code duplication

Reply:
we need to override `update` here because we are updating `current_member`on L10 rather than `current_application`. Or at least, I think it is clearer to override it completely since we are customizing it in multiple ways rather than just in a single way, which is how we usually use ` after_successful_update_hook`. Thoughts?

Reply:
I messed up. Here's the actual code I'm proposing.

```
    def after_successful_update_hook
      current_member.update(step_params)
      update_spouse_of_other_member_if_present
    end

    def application_params
      {}
    end
```